test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/VideoViewTest.cs

```diff
@@ -0,0 +1,135 @@
+﻿
+using global::System;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI.Samples
+{
+    using tlog = Tizen.Log;
+
+    public class VideoViewTest : IExample
+    {
+        public class myPlayer : Tizen.Multimedia.Player
```

Comment:
should we define our own class? is it more natural?

Reply:
Make derived class from Tizen.Multimedia.Player because protected Player(IntPtr handle, Action<int, string> errorHandler)
This constructor's access modifier is protected, so there is no other way.

Reply:
ok!